e2e/tests/app.e2e.js

```diff
@@ -18,15 +27,16 @@ test.describe('AngularJS-Webpack-Babel app', () => {
     await tabsPage.tabThree.click();
```

Comment:
Look, if are going to keep the Page Object helpers methods (such as [clickOnTabThree()][1]), we should at list use them 😛

[1]: https://github.com/xlts-dev/angularjs-webpack-babel/blob/22846b0345f8fb3d243c65a75d31f8388717157f/e2e/pages/tabs-page.js#L26-L28

Reply:
Sorry, I got a little confused. I read a little more regarding Page Object patter. In general, it has 2 main benefits: 
- They keep all page element selectors in one place
- They standardize how tests interact with the page

Tests e2e uses both approaches:
- you can define the click method in PageObject file
- you can invoke `click()` directly in e2e tests.

In our case, there is no difference. If, on the other hand, we had a case where the process of testing a given functionality would be more complicated and not limited to just calling `click()`, then a good practice is to put the whole thing into a separate method in the Page Object file.

I removed click methods from Page Object file and invoke `click()` directly in the e2e tests